includes/admin/discounts/class-discount-codes-table.php

```diff
@@ -189,55 +203,111 @@ protected function get_primary_column_name() {
 	 * @access public
 	 * @since 1.4
 	 *
-	 * @param array $item Contains all the data of the discount code
+	 * @param EDD_Discount $item Discount object.
 	 * @param string $column_name The name of the column
 	 *
 	 * @return string Column Name
 	 */
 	function column_default( $item, $column_name ) {
```

Comment:
since `$item` is an `EDD_Discount` object, can we rename this variable to `$discount` to make it clearer?